File: in_Java/1344 Angle Between Hands of a Clock.java
class Solution {
    public double angleClock(int hour, int minutes) {
        double a = 30.0 * hour + minutes/2.0;
        double b = 6.0 * minutes;
        double c = Math.abs(a - b);
        return Math.min(c, 360.0 - c);
    }
}

File: in_Python/1344 Angle Between Hands of a Clock.py
class Solution(object):
    def angleClock(self, hour, minutes):
        """
        :type hour: int
        :type minutes: int
        :rtype: float
        """
        minute_angle = 6 * minutes
        base = 0
        if hour != 12:
            base = 30 * hour
        hour_angle = base + 30 * minutes / float(60)
        a = abs(hour_angle - minute_angle)
        return min(a, 360-a)
        